app-misc/ola/ola-0.10.7.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```

Comment:
EAPI=7

Reply:
Done. That was not so clear since I saw plenty other ebuilds with the quotes, but most in the tree are without and the Gentoo docs also don't have them

Reply:
Both styles are accepted. EAPI is actually a string and not an integer.